.github/workflows/api-e2e.yml

```diff
@@ -73,6 +73,12 @@ jobs:
              -n 'Admin API E2E 1.0' \
              -o './E2E Tests/Admin-API-Full-Collection.json'
 
+      - name: Combine Collections
```

Comment:
It seems that this step is repeated... it would be better to remove it.

Reply:
No idea how that happened. Maybe after I rebased my branch. I will remove it.

Reply:
Fixed